index.html

```diff
@@ -30,16 +41,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div style="background-color: teal;" class="box">Box 1</div>
+                <div style="background-color: gold;" class="box">Box 2</div>
```

Comment:
Great workaround with the current knowledge you have, however, mixing inline styles with external styling libraries can get messy. Instead of storing your styles in one place, now you have 2.